CONTRIBUTING.adoc

```diff
@@ -65,16 +65,16 @@ caches have not been configured correctly.
 Accepting the configuration settings as outlined above should be sufficient to avoid this, however if your caches are still broken, you may try to append the following lines to `/etc/nix/nix.conf` and/or `~/.config/nix/nix.conf`:
 
 ----
-substituters = https://hydra.iohk.io https://cache.nixos.org/
+substituters = https://cache.iog.io https://cache.nixos.org/
 trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
 extra-experimental-features = nix-command flakes
 ----
 
 On NixOS, set the following NixOS options:
 ----
 nix = {
-  binaryCaches          = [ "https://hydra.iohk.io" "https://cache.iog.io" ];
-  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" ];
+  binaryCaches          = [ "https://cache.iog.io" "https://cache.nixos.org"];
```

Comment:
I think you don't need to do this? the nixos options here are unioned, so the default setting of `cache.iog.io` will not be overwritten. Or am I forgetting?

Reply:
You mean the default settings of `https://cache.nixos.org`? 
In general they are merged yes, so we don't need that. Though it doesn't hurt to keep.
I'll remove it.

Reply:
As someone who knows how nixos works it looked wrong to me, I think it's better to write the idiomatic thing!